server/converters/cxgtool.py

```diff
@@ -10,6 +10,8 @@
     ├─ var                          TileDB array containing gene (column) attributes, with one attribute per
     │                               dataframe column, shape (n_obs,)
     ├─ X                            Main count matrix as a 2D TileDB array, single unnamed numeric attribute
+    ├─ X_col_shift                  TilebDB Array used in column shift encoding, shape (n_var,).
```

Comment:
I would further clarify in the comment (assuming I understand the design): the X_col_shift type must be identical to the X type (ie, float32, int32, etc)

Reply:
Added more comments here.